OpenRA.Mods.Common/Traits/Turreted.cs

```diff
@@ -73,16 +73,28 @@ public void Created(Actor self)
 
 		public virtual void Tick(Actor self)
 		{
-			if (attack != null && !attack.IsAttacking)
+			// NOTE: FaceTarget is called in AttackTurreted.CanAttack if the turret has a target.
```

Comment:
This case isn't really avoidable without doing something worse.

On 2015-06-03, at 6:48 AM, Pavel Penev <notifications@github.com> wrote:

> In OpenRA.Mods.Common/Traits/Turreted.cs:
> 
> > @@ -73,16 +73,28 @@ public void Created(Actor self)
> >  
> >  		public virtual void Tick(Actor self)
> >  		{
> > -			if (attack != null && !attack.IsAttacking)
> > +			// NOTE: FaceTarget is called in AttackTurreted.CanAttack if the turret has a target.
> This is exactly what I meant by
> 
> [21:28:10] well I guess as long as it's not doing anything that would be unexpected...
> 
> That's the definition of "unexpected". :|
> 
> ―
> Reply to this email directly or view it on GitHub.
>